server/helper_test.go

```diff
@@ -321,6 +321,36 @@ func (th *testHelper) SetupRemoteUser(t *testing.T, team *model.Team) *model.Use
 	return user
 }
 
+func (th *testHelper) SetupGuestUser(t *testing.T, team *model.Team) *model.User {
+	t.Helper()
+
+	var user *model.User
+	var err error
+	user = th.SetupUser(t, team)
+
+	user, err = th.p.apiClient.User.UpdateRoles(user.Id, model.SystemGuestRoleId)
+	require.NoError(t, err)
+
+	return user
+}
+
+func (th *testHelper) CreateBot() *model.Bot {
+	id := model.NewId()
+
+	bot := &model.Bot{
+		Username:    "bot" + id,
+		DisplayName: "a bot",
+		Description: "bot",
+		OwnerId:     "ownerID",
+	}
+
+	bot, err := th.p.API.CreateBot(bot)
+	if err != nil {
+		panic(err)
```

Comment:
Can we pass `t *testing.T`, use `t.Helper` and `require.NoError` here instead of panicking?

Reply:
Yes...thanks.